tasks/install.yml

```diff
@@ -31,8 +30,8 @@
 
 - file: dest=/opt/hadoop src={{ hadoop_home }} state=link
 
-- lineinfile: dest="{{ hadoop_home }}/etc/hadoop/hadoop-env.sh" regexp=JAVA_HOME= line="export JAVA_HOME={{item}}"
-  with_first_found:
-   - /usr/lib/jvm/java-8-openjdk-amd64
-   - /usr/lib/jvm/java-7-openjdk-amd64
-   - /usr/lib/jvm/jre-1.7.0
+- lineinfile:
+    dest: "{{ hadoop_home }}/etc/hadoop/hadoop-env.sh"
+    regexp: '^JAVA_HOME='
+    line: 'JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.121-0.b13.el7_3.x86_64/jre'
+
```

Comment:
Ubuntu 14 does not have packages for java 8, so we need to maintain the "with_first_found" to assure it works in it.